test/web-platform-tests/run-to-upstream-web-platform-test.js

```diff
@@ -0,0 +1,86 @@
+"use strict";
```

Comment:
I think that the right way to do would be to fork the web-platform-tests repository, include the tests in the fork and change the web-platform-tests submodule url to this new repository.